File: src/main/java/com/example/dq/dialect/OceanBaseDialect.java
package com.example.dq.dialect;

import com.example.dq.model.DbType;

/** OceanBase(MySQL 模式)方言:元数据与 SQL 语法同 MySQL */
public class OceanBaseDialect extends MySqlDialect {

    @Override
    public DbType type() {
        return DbType.OCEANBASE;
    }

    @Override
    public String driverClassName() {
        return "com.oceanbase.jdbc.Driver";
    }
}

File: dq-desktop/src/main/kotlin/com/example/dq/dialect/OceanBaseDialect.kt
package com.example.dq.dialect

import com.example.dq.model.DbType

/** OceanBase(MySQL 模式)方言:元数据与 SQL 语法同 MySQL */
class OceanBaseDialect : MySqlDialect() {

    override fun type(): DbType {
        return DbType.OCEANBASE
    }

    override fun driverClassName(): String {
        return "com.oceanbase.jdbc.Driver"
    }
}
